packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -186,6 +186,15 @@ static BOOL ShouldSkipFocusZone(NSView *view)
 	return NO;
 }
 
+/// Prevent order change
```

Comment:
Coooulld we have a more descriptive comment here, especially if it's a triple slash, AKA, documentation? :)

Reply:
As an aside, what assumptions can we make here?

Apple's documentation suggests (but doesn't seem to *explicitly* say) that [`accessibilityChildren`](https://developer.apple.com/documentation/appkit/nsaccessibility/1535018-accessibilitychildren) follows the view hierarchy. Is that good enough?

I don't know if we want to make other assumptions about [`accessibilityChildrenInNavigationOrder`](https://developer.apple.com/documentation/appkit/nsaccessibility/2869552-accessibilitychildreninnavigatio), where the docs only say "an easily navigable order," but for the latter I suppose we can lean back on "whatever the OS defaults to."

Reply:
New documentation comment looks good to me 👍

Reply:
The simplicity of the implementation is by-design; I'm not seeking to define "render order" or "UI hierarchy order" any better than "the order of the UI elements as returned by `accessibilityChildren`, which is _an_ order corresponding with the render order and UI hierarchy but certainly not the only order possible.  It gives the RN author the opportunity to change the navigation order from the default calculated order in a predictable way but certainly lacks flexibility if they have a specific problem where neither order is their desired order.  

If someone else requires a different order, we can re-assess the API then with the concrete example; it isn't possible without Fabric for us to have a JSI-callback to allow order to be dynamically determined from JS, and there are flaws with many other possible designs.  If a given order is flawed based on say, the UI tree generated by native components changes the parent-child relationships of native children relative to the VDOM then either the use case is flawed (e.g. `Callout`) or there may be a platform bug worth addressing in the native component